backend/src/config/components/dataInitializer.js

```diff
@@ -0,0 +1,45 @@
+import models from '../../constants/models.js';
+import {
+  ADMIN_PROFILE,
+  ADMIN_USER,
+  MEDICATION1,
+  MEDICATION2,
+  USER_PROFILE,
+  USER_USER,
+} from '../../constants/mockData.js';
+
+const Medication = models[0];
+const Profile = models[2];
+const Settings = models[3];
+const User = models[4];
```

Comment:
It is unsafe - one can add a new model in the first place of this table and everything will be broken.
Please replace array with a object